keyboards/hardwareabstraction/handwire/config.h

```diff
@@ -0,0 +1,12 @@
+// Copyright 2022 Andy Tsai (@atsai)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+#define OLED_FONT_H "custom_fonts.c"
```

Comment:
Just a heads up, this font file looks to be identical to the default font, aside from the logo. 

It may be worth just using the default font, and using `oled_write_raw` to print out the custom logo.